routes/quadkey.js

```diff
@@ -0,0 +1,88 @@
+const ErrorHTTP = require('mapbox-error').ErrorHTTP;
+const db = require('../database/index');
+const Quadkey = db.Quadkey;
+const validator = require('validator');
+const Sequelize = require('sequelize');
+
+module.exports = {
+  getQuadkey,
+  postQuadkey
+};
+
+/**
+ * Gets priority value for a quadkey+project
+ * 
+ * @name get-quadkey-priority
+ * @param {Object} params - Request URL parameters
+ * @param {string} params.quadkey - Quadkey to request priority for
+ * @param {Object} [query] - Request URL query parameters
+ * @param {string} [query.set_id] - Quadkey Set ID to get priority values for
+ */
+function getQuadkey(req, res, next) {
+  const quadkey = req.params.quadkey;
+  const setId = req.query.set_id || null;
+  Quadkey.findOne({
+    where: {
+      quadkey: quadkey,
+      set_id: setId
+    }
+  })
+    .then(quadkey => {
+      res.json({ priority: quadkey.priority });
+    })
+    .catch(() => {
+      return next(
+        new ErrorHTTP('quadkey and set_id combination not found', 404)
+      );
+    });
+}
+
+/**
+ * Write priority values for a quadkey (optionally tied to project)
+ * The backend will handle either INSERTing or UPDATEing as appropriate
+ *
+ * @name post-quadkey-priority
+ * @param {Object} params - Request URL parameters
+ * @param {string} params.quadkey - Quadkey to POST
+ * @param {Object} body - Request body
+ * @param {string|null} body.set_id - Quadkey Set ID or null
```

Comment:
Just to confirm, this body property is required, but if a user does not have a quadkey set ID, they must specify `null`? If this is an optional property, I think you can write this as the following since you're setting the property to `null` if it doesn't exist in the code below:

```js
* @param {string} [body.set_id] - Quadkey Set ID
```